todo_app/app.py

```diff
@@ -71,26 +70,34 @@ def add_item():
     item = request.form.get('item') 
     #Add item to list to do
     add_item_url = "https://api.trello.com/1/cards"
+    add_item_query = buildAddItemParams(item)
+    add_item_resp = query_trello("POST", add_item_url, add_item_query)
+    return redirect(url_for('index'))
+
+def buildAddItemParams(item):
     add_item_query = trello_key_params
```

Comment:
Missed before, but currently you're pointing `add_item_query` at the `trello_key_params` dictionary and then modifying it - which means that you've changed the underlying `trello_key_params` dictionary for next time. It doesn't seem to affect anything here, but it could easily lead to confusion if one of your methods starts passing an argument that wasn't set nearby! Instead if we take an explicit copy of the dictionary then we won't be changing the original

```suggestion
    add_item_query = trello_key_params.copy()
```